i18n/en.json

```diff
@@ -4614,6 +4618,10 @@
     "id": "app.channel.update_last_viewed_at_post.app_error",
     "translation": "Unable to mark channel as unread."
   },
+  {
+    "id": "app.channel.update_member.app_error",
+    "translation": ""
+  },
```

Comment:
Hmm .. this is probably incorrect. I wouldn't expect translations to change/ new ones to be added. Can you check this?

Reply:
oops, yeah. Checked and saw that I missed the unexported `app.updateChannelMember` method. Thanks for the heads up!